websites/L/Leosight/presence.ts

```diff
@@ -0,0 +1,413 @@
+const presence = new Presence({
+    clientId: "888141162488143893"
+  }),
+  time = Math.floor(Date.now() / 1000);
+let item, item2;
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "lslogo",
+      smallImageKey: "lsminilogo",
+      smallImageText: "leosight.cz"
+    },
+    path = document.location.pathname;
+  presenceData.startTimestamp = time;
```

Comment:
```suggestion
      smallImageText: "leosight.cz",
      startTimestamp: time
    },
    path = document.location.pathname;
```

Reply:
then I'll be able to remove it from everywhere without changing the function?

Reply:
yes?

Reply:
K, thanx